docs/rackhd/api-usage-examples.rst

```diff
@@ -45,7 +50,7 @@ This example provides instructions on how to flash a BMC image on a Quanta (node
    The above command will return a SKU ID which will be used to post the Workflow against the node. If an error like "Duplicate entry
    exists" is returned in place of the the SKU ID, check the database and delete a prexisting SKU package.
```

Comment:
don't delete, maybe use PUT method to replace the original sku pack is a better way.